app/Listeners/Stand/SendStandAllocationAcarsMessage.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace App\Listeners\Stand;
+
+use App\Acars\Exception\AcarsRequestException;
+use App\Acars\Message\Telex\StandAssignedTelexMessage;
+use App\Acars\Provider\AcarsProviderInterface;
+use App\Events\StandAssignedEvent;
+use App\Models\Stand\StandAssignment;
+use App\Services\LocationService;
+use Illuminate\Support\Facades\Log;
+use Location\Distance\Haversine;
+
+class SendStandAllocationAcarsMessage
+{
+    private const MIN_ASSIGNMENT_DISTANCE_NAUTICAL_MILES = 7.5;
+
+    private readonly AcarsProviderInterface $acarsProvider;
+
+    public function __construct(AcarsProviderInterface $acarsProvider)
+    {
+        $this->acarsProvider = $acarsProvider;
+    }
+
+    public function handle(StandAssignedEvent $assignedEvent)
+    {
+        if (!$this->standAcarsMessagesAreEnabled()) {
+            return;
+        }
+
+        $standAssignment = $assignedEvent->getStandAssignment();
+        if (!$this->userAllowsStandAllocationMessages($standAssignment)) {
+            return;
+        }
+
+        if (!$this->meetsSendingConditions($standAssignment)) {
+            return;
+        }
+
+        try {
+            $this->acarsProvider->sendTelex(new StandAssignedTelexMessage($standAssignment));
+        } catch (AcarsRequestException $requestException) {
+            Log::error(
+                sprintf('Acars exception sending stand allocation message: %s', $requestException->getMessage())
+            );
+        }
+    }
+
+    private function meetsSendingConditions(StandAssignment $assignment): bool
```

Comment:
Do we want to include only when an airport has a minimum of GMC too?

Reply:
I feel like it could be come the wild west if we don't restrict it to when the airport is controlled

Reply:
I was thinking about this - perhaps we should make it a user option? So people who want to be given a stand at uncontrolled airfields can opt in?

Reply:
Yeah that seems reasonable. They have to opt-in anyway so adding that extra option for those who want extra realism.

Reply:
Done this in https://github.com/VATSIM-UK/uk-controller-api/pull/1298/commits/edb9ac1585897c274f7ea9dc4e2dd9d456fed1bb